ui/app/templates/rule_modal.html

```diff
@@ -153,6 +153,15 @@ <h3 class="modal-title" ng-show="!is_edit && is_duplicate">Duplicate Existing Ru
         </div>
       </div>
     </div>
+    <div class="row">
```

Comment:
This field also needs to autocomplete the mapping names, like the Mapping field does. Please avoid making an extra network request for this if possible (ie: mapping and fallbackMapping should be able to share the same data).